src/screens/ExchangeModal.tsx

```diff
@@ -130,13 +143,47 @@ const getShowOutputField = type => {
   }
 };
 
+interface Asset {
```

Comment:
@estebanmino @jxom Ive added this type as I couldnt find anything that matched it in the codebase. If we want it where would be the best place to have it for future reference?

Reply:
ideally inside `entities` folder